CRITICAL - 2023-01-17 01:35:33 --> Class "Tests\Support\Entity\User" not found
#0 C:\xampp\htdocs\cdi\system\CodeIgniter.php(832): App\Controllers\Home->index()
#1 C:\xampp\htdocs\cdi\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#2 C:\xampp\htdocs\cdi\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\cdi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\cdi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2023-01-17 01:35:33 --> Class "Tests\Support\Entity\User" not found
#0 C:\xampp\htdocs\cdi\system\CodeIgniter.php(832): App\Controllers\Home->index()
#1 C:\xampp\htdocs\cdi\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#2 C:\xampp\htdocs\cdi\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\cdi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\cdi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
ERROR - 2023-01-17 04:04:13 --> Error connecting to the database: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée
CRITICAL - 2023-01-17 04:04:13 --> Unable to connect to the database.
Main connection [MySQLi]: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée
#0 C:\xampp\htdocs\cdi\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\cdi\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 C:\xampp\htdocs\cdi\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\cdi\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('hacker@mail.com', 'email', Array, NULL)
#4 C:\xampp\htdocs\cdi\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'hacker@mail.com', Array, Array)
#5 C:\xampp\htdocs\cdi\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 C:\xampp\htdocs\cdi\app\Controllers\Home.php(51): CodeIgniter\Controller->validate(Array)
#7 C:\xampp\htdocs\cdi\system\CodeIgniter.php(832): App\Controllers\Home->inserted()
#8 C:\xampp\htdocs\cdi\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\cdi\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\cdi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\cdi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2023-01-17 04:20:30 --> Error connecting to the database: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée
CRITICAL - 2023-01-17 04:20:30 --> Unable to connect to the database.
Main connection [MySQLi]: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée
#0 C:\xampp\htdocs\cdi\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\cdi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#2 C:\xampp\htdocs\cdi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\cdi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\cdi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#5 C:\xampp\htdocs\cdi\app\Controllers\Home.php(33): CodeIgniter\Model->insert(Array)
#6 C:\xampp\htdocs\cdi\system\CodeIgniter.php(832): App\Controllers\Home->inserted()
#7 C:\xampp\htdocs\cdi\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\cdi\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\cdi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\cdi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#11 {main}
